src/containers/TrendContainer.js

```diff
@@ -96,11 +98,26 @@ TrendContainer.propTypes = {
   until: PropTypes.number.isRequired,
 }
 
-const mapStateToProps = ({ filters, summaries }) => ({
-  ...filters,
-  ...getPlaceInfo(filters),
-  summaries,
-})
+export const mapStateToProps = ({ filters, summaries }) => {
```

Comment:
this change feels overly involved -- we don't need to change the `summaries.data` object that the component gets, we just need a better way to define the `places` variable (which i would do in the `mapStatetoProps`), and then the render function gets even simpler

```
const places = [place]
if (place !== nationalKey) places.push(nationalKey)

return {
  ...filters,
  ...getPlaceInfo(filters),
  summaries,
  places,
}
```